test/Npgsql.Tests/TestUtil.cs

```diff
@@ -141,6 +141,22 @@ public static object ExecuteScalar(this NpgsqlConnection conn, string sql, Npgsq
         }
     }
 
+    public static class NpgsqlCommandExtensions
+    {
+        public static T ExecuteScalar<T>(this NpgsqlCommand cmd)
+        {
+            using (var rdr = cmd.ExecuteReader())
+                return rdr.Read() ? rdr.GetFieldValue<T>(0) : default;
+        }
+
+        public static NpgsqlDataReader ExecuteRecord(this NpgsqlCommand cmd)
+        {
+            var rdr = cmd.ExecuteReader();
+            rdr.Read();
```

Comment:
Probably a good idea to assert that `Read()` returned true

Reply:
A good idea, will do.